packages/frontend/src/components/buy/FundingType.jsx

```diff
@@ -45,9 +45,9 @@ const WrapperImg = styled.div`
     display: flex;
 `;
 
-export const FundingType = ({ icon, link, name, track }) => {
+export const FundingType = ({ icon, link, name, track, ref }) => {
     return (
-        <LinkWrap href={link} target='_blank' rel='noreferrer' onClick={() => track && track()}>
+        <LinkWrap href={link} target='_blank' rel={!ref ? 'noreferrer' : null} onClick={() => track && track()}>
```

Comment:
@samkrew have you tested this?  I'm not sure if this will work as you intend.
https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types

Reply:
Fixed here: https://github.com/near/near-wallet/pull/2737/commits/29708baefe8a53635599d15ee35c741e6abd564f